src/lib/multiSemanticRelease.ts

```diff
@@ -93,14 +93,18 @@ export default async function multiSemanticRelease(
   try {
     const batches = batchingToposort(pkgDag)
     for (const batch of batches) {
+      const promises: Array<Promise<void>> = []
       for (const pkgName of batch) {
         const pkg = packages.find(_pkg => _pkg.name === pkgName)
         if (!pkg) {
           throw new Error(`Inexistant package ${pkgName} in batch`)
         }
 
-        await releasePackage(pkg, createInlinePlugin, multiContext, flags)
+        promises.push(
+          releasePackage(pkg, createInlinePlugin, multiContext, flags),
+        )
```

Comment:
Or maybe we can add an option to enable/disable concurrency? This way we can easily test/rollback if we got issue? 
If I remember well, concurrency caused flaky issues back then so we would maybe need to test it for few weeks

Reply:
Oue bien sur, bonne idee